Description: Makefile fixes
 The upstream Makefile is extremely conservative, presumably to support
 certain esoteric operating systems and architectures. This patch makes
 some minor edits to permit the package to play nice with debhelper et al.
Author: Tom Lee <debian@tomlee.co>
Forwarded: no
Last-Update: 2020-04-25
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Src/makefile
+++ b/Src/makefile
@@ -14,7 +14,7 @@
 	# debugging:  make CFLAGS=-pg -g -DNXT
 
 # for a more picky compilation use gcc-4 and add:
-PICKY=-std=c99 -Wstrict-prototypes -pedantic -fno-strength-reduce \
+CFLAGS+=-std=c99 -Wstrict-prototypes -pedantic -fno-strength-reduce \
 	-fno-builtin -W -Wshadow -Wpointer-arith \
 	-Wcast-qual -Winline -Wall
 
@@ -23,7 +23,7 @@
 			# for Visual C++: spin -PCL  -E/E
 
 YACC?=yacc		# on Solaris: /usr/ccs/bin/yacc
-YFLAGS=-v -d 		# creates y.output and y.tab.h
+YFLAGS?=-v -d 		# creates y.output and y.tab.h
 DESTDIR?=/usr/local
 INSTALL?=cp		# on linux: install -D
 
